File: app/src/main/java/com/hashim/mapswithgeofencing/Models/WeatherModel/ListWeatherModel/Clouds.java
/*
 * Copyright (c) 2021/  3/ 20.  Created by Hashim Tahir
 */

package com.hashim.mapswithgeofencing.Models.WeatherModel.ListWeatherModel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Clouds {

    @SerializedName("all")
    @Expose
    private Integer all;

    public Integer getAll() {
        return all;
    }

    public void setAll(Integer all) {
        this.all = all;
    }

}

File: app/src/main/java/com/hashim/mapswithgeofencing/network/response/weather/Clouds.kt
/*
 * Copyright (c) 2021/  3/ 25.  Created by Hashim Tahir
 */

package com.hashim.mapswithgeofencing.network.response.weather


import com.google.gson.annotations.SerializedName

data class Clouds(
        @SerializedName("all")
        val all: Int
)